src/actions/article.js

```diff
@@ -49,4 +54,36 @@ const getArticle = (articleSlug) => (dispatch) => {
     });
 };
 
-export default getArticle;
+
+// Get all articles
+
+const fetchAllArticleSuccess = payload => ({
+  type: FETCH_ARTICLES_SUCCESS,
+  payload
+});
+
+const fetchAllArticleLoading = payload => ({
+  type: FETCH_ARTICLES_LOADING,
+  payload
+});
+
+const fetchAllArticleError = payload => ({
+  type: FETCH_ARTICLES_ERROR,
+  payload
+});
+
+// Get all articles
+export const getAllArticle = () => (dispatch) => {
+  dispatch(fetchAllArticleLoading(true));
+  return axios
+    .get(`${API}/api/articles`)
+    .then(response => {
+      console.log(response);
+      if (response.data.status === 'success') {
+        dispatch(fetchAllArticleLoading(false))
+        return dispatch(fetchAllArticleSuccess(response.data.articles));
+      }
+      return dispatch(fetchAllArticleError('some erroe here'));
```

Comment:
I think you used lines 86 and 88 for debugging, do check it and replace with error from request response